_sass/components/_sticky_nav.scss

```diff
@@ -2,16 +2,19 @@
   position: absolute;
   left: percentage(8 / 12);
   padding-left: $base-padding-extra;
-  top: 0;
+  top: 0;  
+  @include respond-to(medium-down) {
+     display: none !important;
```

Comment:
!important should not be used<br>Line should be indented 4 spaces, but was indented 5 spaces